foxpuppet/tests/conftest.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+from foxpuppet.foxpuppet import FoxPuppet
```

Comment:
Maybe make it available via __init__.py so that a 'from foxpuppet import FoxPuppet' would be enough.